components/widgets/menu/menu_header.scss

```diff
@@ -6,15 +6,15 @@
         clear: both;
         color: inherit;
         cursor: default;
-        display: block;
+        display: flex;
```

Comment:
This should be handled by component SCSS, as well, since changing the base-component will certainly have side-effects

Reply:
++ @asaadmahmood